scripts/dscat.py

```diff
@@ -1,110 +0,0 @@
-#!/usr/bin/env python
```

Comment:
This is technically a  separate implementation in python 2. You wouldn't have to remove it for the CAF upgrade, but I'll accept it anyways because python 2 is EOLd and i don't think I've ever seen anybody use this.